demo/openssl-qat-engine/Dockerfile

```diff
@@ -1,14 +1,26 @@
-ARG FINAL_BASE_IMAGE=clearlinux:base
-
-FROM clearlinux:base as builder
+FROM ubuntu:21.10 as builder
```

Comment:
Any reason to use ubuntu 21.10 for this demo? Can we use LTS (20.04/focal)?

Reply:
no particular reason. My thinking was "as new as possible" but I'll move to `focal`